legend-sdlc-server/src/test/java/org/finos/legend/sdlc/server/gitlab/api/TestBaseGitLabApi.java

```diff
@@ -0,0 +1,91 @@
+        Assert.assertTrue(BaseGitLabApi.isValidEntityPath("model::test::function::test_String_$1_MANY$__String_1_"));
+
+        Assert.assertFalse(BaseGitLabApi.isValidEntityPath("model"));
+        Assert.assertFalse(BaseGitLabApi.isValidEntityPath("model::"));
+        Assert.assertFalse(BaseGitLabApi.isValidEntityPath("model::*"));
+        Assert.assertFalse(BaseGitLabApi.isValidEntityPath("model::$test::$function"));
+        Assert.assertFalse(BaseGitLabApi.isValidEntityPath("meta::test::ValidClassName"));
+        Assert.assertFalse(BaseGitLabApi.isValidEntityPath("model::has::other::characters::test_String_$1_10$__String_$1_*$_&"));
+    }
+
+    @Test
+    public void testIsValidPackagePath()
+    {
+        Assert.assertTrue(BaseGitLabApi.isValidPackagePath("model"));
+        Assert.assertTrue(BaseGitLabApi.isValidPackagePath("random::entity::path"));
+
+        Assert.assertFalse(BaseGitLabApi.isValidPackagePath("model::"));
+        Assert.assertFalse(BaseGitLabApi.isValidPackagePath("model::*"));
+        Assert.assertFalse(BaseGitLabApi.isValidPackagePath("model::$test::$function"));
+        Assert.assertFalse(BaseGitLabApi.isValidPackagePath("model::test::function::test_String_$1_10$__String_MANY_"));
+    }
+
+    @Test
+    public void testIsValidPackageableElementPath()
+    {
+        Assert.assertTrue(BaseGitLabApi.isValidPackageableElementPath("model"));
+        Assert.assertTrue(BaseGitLabApi.isValidPackageableElementPath("meta::entity::path"));
+        Assert.assertTrue(BaseGitLabApi.isValidPackageableElementPath("random::entity::path"));
+        Assert.assertTrue(BaseGitLabApi.isValidPackageableElementPath("model::test::function::test_String_$1_10$__String_MANY_"));
+
+        Assert.assertFalse(BaseGitLabApi.isValidPackageableElementPath("model::"));
+        Assert.assertFalse(BaseGitLabApi.isValidPackageableElementPath("model::*"));
+        Assert.assertFalse(BaseGitLabApi.isValidPackageableElementPath("model::$test::$function"));
+        Assert.assertFalse(BaseGitLabApi.isValidPackageableElementPath("model::has::other::characters::test_String_$1_10$__String_$1_*$_&"));
+    }
+
+    @Test
+    public void testIsValidClassifierPath()
+    {
+        Assert.assertTrue(BaseGitLabApi.isValidClassifierPath("meta::entity::path"));
+        Assert.assertTrue(BaseGitLabApi.isValidClassifierPath("meta::pure::metamodel::function::ConcreteFunctionDefinition"));
+        Assert.assertTrue(BaseGitLabApi.isValidClassifierPath("meta::entity::path::test_String_$1_10$__String_MANY_"));
+
+        Assert.assertFalse(BaseGitLabApi.isValidClassifierPath("meta::"));
+        Assert.assertFalse(BaseGitLabApi.isValidClassifierPath("meta::*"));
+        Assert.assertFalse(BaseGitLabApi.isValidClassifierPath("meta::$test::$function"));
+        Assert.assertFalse(BaseGitLabApi.isValidClassifierPath("meta::has::other::characters::test_String_$1_10$__String_$1_*$_&"));
```

Comment:
Can you add one or more asserts here that show that a classifier path that starts with something other than "meta::" is not valid?